docker-compose-junit-jupiter/build.gradle

```diff
@@ -0,0 +1,15 @@
+apply plugin: 'org.unbroken-dome.test-sets'
+
+testSets {
+    integrationTest
+}
+
+build.dependsOn integrationTest
+
+dependencies {
+    compile project(':docker-compose-rule-core')
+    compile 'org.junit.jupiter:junit-jupiter'
```

Comment:
can we just use junit-jupiter-api?
https://mvnrepository.com/artifact/org.junit.jupiter/junit-jupiter-api